src/reducers/index.js

```diff
@@ -3,10 +3,12 @@ import alert from "./alertReducer"
 import auth from "./authReducer"
 import user from "./userReducer"
 import quiz from "./quizReducer"
+import mindset from "./mindsetReducer"
 
 export default combineReducers({
   alert,
   auth,
   user,
-  quiz
+  quiz,
+  mindset,
 })
```

Comment:
Boy it took me forever to remind myself that this is where I add new reducers...

Reply:
Yep - this has caused many headaches.